README.md

```diff
@@ -83,6 +83,8 @@ requests there.
 
 * For git commit messages use the rules stated on
   [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/) as a reference
+* As a SUSE colleague consider signing commits which we consider to use for
+  automatic deployments within SUSE
```

Comment:
It's not quite clear to me what this means. Assuming contributions from non-SUSE contributors get merged, they could not be auto-deployed. Ok. But if i as a SUSE employee commit something afterwards, would that mean i have to vouch for all previous commits with my signature, because they can now be auto-deployed?

Reply:
Well, signing is built into git. Signing a commit means signing the hash which is unique considering the history meaning effectively you "sign the complete history leading up to that". But still the owners of the deployment need to take responsibility for how they deploy. So short answer is: yes, but no harm will come to you :)

Reply:
To me it changes how much weight signing a commit has. I'm not just responsible for the deployment of my own contribution, but also for all the automatically merged Dependabot updates (since the last signed commit).

Reply:
At first i just wanted to argue that the extra responsibility should be mentioned in the documentation. But the more i think about it the more i believe that only SUSE employees that have opted into auto-deployment of their commits should actually be included here.

Reply:
We already have the same rule in qem-bot and again I don't see the problem: if SUSE employees sign with their key and put the same key on the deployment instance then and only then we trust the repo up to the last commit signed that way. That's better than unconditionally deploying the current state in git. If you have recommendations for rephrasing please say so.